citation/management/commands/remove_orphans.py

```diff
@@ -12,9 +10,19 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         qs = models.Platform.objects.exclude(pk__in=models.Publication.platforms.through.objects.values('platform'))
+        print("              -----------------------------------------------------------------           ")
```

Comment:
Please use `logger.info` here instead of printing to stdout. 

Also use `join` instead of printing each thing, e.g,. `logger.info("%s", '\n'.join(qs))` or something like that (untested)